test/azure/terraform_azure_postgresql_example_test.go

```diff
@@ -1,3 +1,8 @@
+// +build azure
```

Comment:
Missing build tag on our Azure code. Not related to this PR, but was causing test failures.